cli/src/pcluster/validators/budget_validators.py

```diff
@@ -0,0 +1,62 @@
+from time import sleep
+
+import boto3
+
+from pcluster.validators.common import FailureLevel, Validator
+
+
+class BudgetFilterTagValidator(Validator):
+    """
+    Budget CostFilters Tag validator.
+
+    Validate if the tag used for the filter is active under the user's account.
+    """
+
+    def _validate(self, budget_category):
+        if budget_category == "cluster":
```

Comment:
nit:
we can create a map to store the budget_category_and_tag_key_map
```
budget_category_and_tag_key_map = {"cluster": "parallelcluster:cluster-name", "queue": "parallelcluster:queue-name"}
key = budget_category_and_tag_key_map.get(budget_category, None)
if not key:
   return
```
This can avoid having too many “if else” when the budget_category expand

Reply:
Good idea. Implemented!